### PR TITLE
fix: ng install after prow config refactoring

### PR DESCRIPTION
**Pull request by ccojocar**
#### Submitter checklist

- [ ] Change is code complete and matches issue description.
- [ ] Change is covered by existing or new tests.

#### Which issue this PR fixes

fixes #

### CODE REVIEW

File: pkg/jx/cmd/opts/install.go [hunk 1]
```diff
@@ -1557,7 +1556,7 @@ func (o *CommonOptions) GetClusterUserName() (string, error) {
 
 	config, _, err := o.Kube().LoadConfig()
 	if err != nil {
-		return username, err
+		return username, errors.Wrap(err, "lodaing kube config")
```
Comment (hferentschik) [inline pkg/jx/cmd/opts/install.go]: typo "loading"

File: pkg/prow/prow.go [hunk 2]
```diff
@@ -607,11 +622,11 @@ func CreateProwJob(client kubernetes.Interface, ns string, j prowapi.ProwJob) (p
 	retJob := prowapi.ProwJob{}
 	body, err := json.Marshal(j)
 	if err != nil {
-		return retJob, err
+		return retJob, errors.Wrap(err, "marshaling the prow job")
 	}
 	resp, err := client.CoreV1().RESTClient().Post().RequestURI(fmt.Sprintf("/apis/prow.k8s.io/v1/namespaces/%s/prowjobs", ns)).Body(body).DoRaw()
 	if err != nil {
-		return retJob, fmt.Errorf("failed to create prowjob %v: %s", err, string(resp))
+		return retJob, fmt.Errorf("createing prowjob %v: %s", err, string(resp))
```
Comment (hferentschik) [inline pkg/prow/prow.go]: "creating"

File: pkg/prow/prow.go [hunk 3]
```diff
@@ -607,11 +622,11 @@ func CreateProwJob(client kubernetes.Interface, ns string, j prowapi.ProwJob) (p
 	retJob := prowapi.ProwJob{}
 	body, err := json.Marshal(j)
 	if err != nil {
-		return retJob, err
+		return retJob, errors.Wrap(err, "marshaling the prow job")
```
Comment (hferentschik) [inline pkg/prow/prow.go]: "marshalling"?! Maybe both is possible?

File: pkg/prow/prow.go [hunk 4]
```diff
@@ -577,17 +593,16 @@ func (o *Options) GetReleaseJobs() ([]string, error) {
 }
 
 func (o *Options) GetPostSubmitJob(org, repo, branch string) (config.Postsubmit, error) {
-
 	p := config.Postsubmit{}
 	cm, err := o.KubeClient.CoreV1().ConfigMaps(o.NS).Get(ProwConfigMapName, metav1.GetOptions{})
 	if err != nil {
-		return p, err
+		return p, errors.Wrapf(err, "getting the presugmit jobs from %q config map in namespace %q", ProwConfigMapName, o.NS)
```
Comment (hferentschik) [inline pkg/prow/prow.go]: "presugmit" ?

File: pkg/jx/cmd/opts/common.go [hunk 5]
```diff
@@ -219,8 +219,9 @@ func (o *CommonOptions) KubeClient() (kubernetes.Interface, error) {
 			return nil, err
 		}
 		o.kubeClient = kubeClient
-		o.currentNamespace = currentNs
-
+		if o.currentNamespace == "" {
+			o.currentNamespace = currentNs
```
Comment (hferentschik) [inline pkg/jx/cmd/opts/common.go]: So this is the actual "fix", right? The rest is improving on error reporting, right?

Comment (ccojocar) [reply pkg/jx/cmd/opts/common.go]: right

**Pull request merged**